### PR TITLE
Update fr-Fr.json

### PR DESCRIPTION
**Pull request by Thomasfda**

### CODE REVIEW

**Comment by Thomasfda** [comment]
Update ready.

File: www/lang/fr-Fr.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
         "wrongUsernameOrPassword": "Nom d'utilisateur ou mot de passe incorrect!",
         "failedToLogIn":  "Échec d'identification", 
         "userNotValid": "Nom d'utilisateur invalide!"
+        "loggingIn": "Connexion en cours..."
```
Comment (filfat) [inline www/lang/fr-Fr.json]: You forgot to add a , at the end of the previous line :)

**Comment by Thomasfda** [comment]
I have add , at the end of previous line :)

**Comment by filfat** [comment]
Awesome

**Pull request merged**